radicle-httpd/src/api/v1/delegates.rs

```diff
@@ -0,0 +1,70 @@
+use axum::response::IntoResponse;
+use axum::routing::get;
+use axum::{Extension, Json, Router};
+use serde::Serialize;
+
+use radicle::cob::store::Store;
+use radicle::git::Oid;
+use radicle::identity::project::Payload;
+use radicle::identity::{Did, Doc};
+use radicle::storage::{ReadRepository, WriteStorage};
+
+use crate::api::axum_extra::Path;
+use crate::api::error::Error;
+use crate::api::Context;
+
+/// Project info.
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Info {
+    /// Project metadata.
+    #[serde(flatten)]
+    pub payload: Payload,
```

Comment:
this struct is moved here, previously it was [here](https://github.com/radicle-dev/radicle-client-services/blob/master/http-api/src/project.rs#L20) and `head` was an `Option`, but [`repo.head()` ](https://github.com/radicle-dev/heartwood/pull/113/files#diff-8489cd4775842a934f20ffe06bc2e477f3e21b887fbb4573a88208cd08c4a6ccR49) no longer returns an `Option` so it was changed.